liquibase/scripts/generate_liquibase_snapshots.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+#
+#  This Source Code Form is subject to the terms of the Mozilla Public License,
+#  v. 2.0. If a copy of the MPL was not distributed with this file, You can
+#  obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+#  the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+#
+#  Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+#  graphic logo is a trademark of OpenMRS Inc.
+#
+
+
+function generate_liquibase_snapshots_and_update() {
+	cd liquibase/
+	. scripts/create_liquibase_snapshots.sh "${1}" "${2}"
+	. scripts/fix_liquibase_snapshots.sh
+	cd ..
+}
+
+function move_updated_snapshots() {
+    mv liquibase/snapshots/liquibase-core-data-UPDATED-SNAPSHOT.xml api/src/main/resources/org/openmrs/liquibase/snapshots/core-data/"liquibase-core-data-$openmrs_version.x.xml"
+    mv liquibase/snapshots/liquibase-schema-only-UPDATED-SNAPSHOT.xml api/src/main/resources/org/openmrs/liquibase/snapshots/schema-only/"liquibase-schema-only-$openmrs_version.x.xml"
+}
+
+function create_new_liquibase_update_file() {
+    new_liquibase_file="liquibase-update-to-latest-${new_openmrs_version}.x.xml"
+    cp "api/src/main/resources/liquibase-update-to-latest-template.xml" "api/src/main/resources/org/openmrs/liquibase/updates/$new_liquibase_file"
+    
+    database_updater_class="api/src/test/java/org/openmrs/util/DatabaseUpdaterDatabaseIT.java"
+    variable_name="CHANGE_SET_COUNT_FOR_GREATER_THAN_2_1_X"
```

Comment:
Why are we explicitly using 2.1.x here?

Reply:
according to the steps mentioned [here](https://github.com/openmrs/openmrs-core/tree/master/liquibase#step-1---add-the-new-snapshot-files), we are copying the liquibase-updates template and rename it to current version. That template file contains change set as below (this include a comment), 
 `<changeSet id="42ddb873-f064-4418-a6c0-6c09426c4832" author="wschlegel">
		<comment>Empty change set for integration tests.</comment>
	</changeSet>` 

So when adding a new change-set we need to increase the `CHANGE_SET_COUNT_FOR_GREATER_THAN_2_1_X` value by one.

Reply:
But the current version is not 2.1.?

Reply:
when adding new `changeset` in a new version which is greater than 2.1, we should increase this `CHANGE_SET_COUNT_FOR_GREATER_THAN_2_1_X` value by the number of newly added changesets.